packages/common/src/store/buy-usdc/slice.ts

```diff
@@ -59,6 +58,7 @@ const slice = createSlice({
     buyUSDCFlowFailed: (state) => {
       // TODO: Probably want to pass error in action payload
       state.error = new Error('USDC purchase failed')
+      state.stage = BuyUSDCStage.ERROR
```

Comment:
Don't we want to know what stage the error happened in?

Reply:
oops you are correct